scripts/README.md

```diff
@@ -1,15 +1,21 @@
-# Scripts: Scripts for variosu Sparkly things
+# Scripts: Scripts for various Sparkly things
 
-Most scripts have usage instructions.
+Most scripts have their own usage instructions within the script themselves.
 
-However some of these scripts require a file called `prodAccountKey.json`.
-
-To obtain this file, go to Firebase and follow these steps:
+These scripts require a Firebase account key, which you setup once for each environment. To obtain this file, go to Firebase and follow these steps:
 
 1. Go to Project Settings
 2. Go to Service Accounts
 3. Click `Generate new private key`
-4. Save the key
-5. Move it to `scripts/prodAccountKey.json`
+4. You will be prompted to download a file `PROJECT-ID-firebase-adminsdk-abc12-1234567890.json`
+5. Save it to your root `scripts` folder (it is automatically excluded in our default .gitignore file)
+
+WARNING - this file grants passwordless access to ALL data in Firebase, including delete access. Treat the file like a private key and keep it safe.
+
+Run your chosen script just like any other command line tool. If you are creating a new environment, we recommend starting with the Bootstrap script:
+
+`./bootstrap-new-environment.ts [project-id] [project-private-key]`
+
+For example:
 
-WARNING - this file grants passwordless access to ALL data in Firebase including delete access. Treat the file like a private key and keep it safe.
+`./bootstrap-new-environment.ts my-project my-project-firebase-adminsdk-abc12-1234567890.json`
```

Comment:
Triple backticks

Reply:
like this:

<pre><code>```bash
./bootstrap-new-environment.ts my-project my-project-firebase-adminsdk-abc12-1234567890.json
```</code></pre>

Reply:
@sunny-viktoryia i don’t actually see a difference when I look at the preview... what’s the diff, yo???